packages/react-instantsearch/src/connectors/connectConfigure.js

```diff
@@ -13,22 +16,25 @@ export default createConnector({
     return searchParameters.setQueryParameters(items);
   },
   transitionState(props, prevSearchState, nextSearchState) {
```

Comment:
I still want :) 
Needs more work on the Configure side. Would be done I hope, soon, with another PR.